dsl/src/main/scala/com.crobox.clickhouse/dsl/language/ClickhouseTokenizerModule.scala

```diff
@@ -64,19 +64,21 @@ trait ClickhouseTokenizerModule
 
   private[language] def toRawSql(query: InternalQuery)(implicit ctx: TokenizeContext): String =
     query match {
-      case InternalQuery(select, from, as, prewhere, where, groupBy, having, join, orderBy, limit, union) =>
+      case InternalQuery(select, from, prewhere, where, groupBy, having, join, orderBy, limit, union) =>
         s"""
            |${tokenizeSelect(select)}
            | ${tokenizeFrom(from)}
-           | ${tokenizeAs(as)}
            | ${tokenizeJoin(select, from, join)}
            | ${tokenizeFiltering(prewhere, "PREWHERE")}
            | ${tokenizeFiltering(where, "WHERE")}
            | ${tokenizeGroupBy(groupBy)}
            | ${tokenizeFiltering(having, "HAVING")}
            | ${tokenizeOrderBy(orderBy)}
            | ${tokenizeLimit(limit)}
-           | ${tokenizeUnionAll(union)}""".trim.stripMargin.replaceAll("\n", "").replaceAll("\r", "")
+           | ${tokenizeUnionAll(union)}""".trim.stripMargin
+          .replaceAll("\n", "")
+          .replaceAll("\r", "")
+          .replaceAll(" +", " ") // replace double (or more) subsequent spaces by one
```

Comment:
What if i have a query that looks for `description = ' a  b '`? (ironcly it collapses the spaces in github editor) I think we should not replace any space, \n or \r in this phase because it might affect user input